index.html

```diff
@@ -20,60 +24,113 @@
+          </div>
+        </div>
+  </section>
+
+    <section>
+        <div class="container mt-3">
+            <h2>Curious about what I can do for you? Here's a few of my past work</h2>
+        </div>
+    </section>
+
+    <section>
+        <div class="container text-center">
+            <div class="row flex-column flex-md-row">
+              <div class="col">
+                <figure class="figure">
+                    <div class="btn-relative">
+                        <img src="./assets/ladieslounge.png" class="figure-img img-fluid rounded img-project-home" alt="...">
+                        <a href="work/ladies-lounge.html"><button type="button" class="btn btn-color btn-absolute btn-shadow">Check out the full project</button></a>
+                    </div>
+                    <figcaption class="figure-caption">Every good brand starts with a logo and a brand identity that represent who they are. Here's a project I did for a brand called The Ladies' Lounge.</figcaption>
+                </figure>
+              </div>
+
+              <div class="col">
+                <figure class="figure">
+                    <div class="btn-relative">
+                        <img src="./assets/roseanna.jpg" class="figure-img img-fluid rounded img-project-home" alt="...">
+                        <a href="work/roseanna-skincare.html"><button type="button" class="btn btn-color btn-absolute btn-shadow">Check out the full project</button></a>
+                    </div>
+                    <figcaption class="figure-caption">Showcase your services or products with a well-designed, personalized website. Here's one I did for a skincare brand named RoseAnna.</figcaption>
+                </figure>
+              </div>
+
+              <div class="col">
+                <figure class="figure">
+                    <div class="btn-relative">
+                        <img src="./assets/bearpaws_blue.PNG" class="figure-img img-fluid rounded img-project-home" alt="...">
+                        <a href="work/bear-paws.html"><button type="button" class="btn btn-color btn-absolute btn-shadow">Check out the full project</button></a>
                     </div>
-                </div>
+                    <figcaption class="figure-caption">Lettering is a great tool to convey a very personalized message to your audience. Here's a project I did for Bear Paws.</figcaption>
+                </figure>
+              </div>
+            </div>
+          </div>
+    </section>
+    
+    <footer class="footer-background text-white">
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-12">
+              <p class="text-center">&copy; Marie-Pier Dubois, 2023</p>
+              <ul class="list-inline text-center">
+                <li class="list-inline-item"><a href="https://www.facebook.com/mpdubois11"><img src="./assets/Réseaux sociaux/facebook-with-circle-01.svg" alt="Facebook"></a></li>
```

Comment:
External links generally should have `target='_blank'` property to open the page in a new tab. Do the same for all external links.